server/Core/Entities/Competition.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Core.Entities;
+
+public class Competition
+{
+    public int CompetitionId { get; set; }
+    public string? Name { get; set; }
+    public DateTime? WeightingDate { get; set; }
+    public DateTime? StartDate { get; set; }
+    public DateTime? EndDate { get; set; }
+    public string? City { get; set; }
+    public string? CompetitionLevel { get; set; }
```

Comment:
Can we have a competition level table? Would be easier to manage instead of strings I guess

Reply:
I had plans to do it later. I think strings are enough for now.